src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
I'll be happy to participate in a thread on this issue, but yes, we did exchange emails.

I want to (soon) build a prototype to show what I think is possible... I'd be happy to do so publicly.